Plugin/Energistics/Mapping/WitsmlWellboreCompletionPerforationToVtkPolyData.cxx

```diff
@@ -102,10 +104,10 @@ void WitsmlWellboreCompletionPerforationToVtkPolyData::loadVtkObject()
 
 			// Get the MD and XYZ values for all the patches in the wellbore trajectory.
 			const uint64_t pointCount = this->wellboreTrajectory->getXyzPointCountOfAllPatches(); // std::vector<double> mdValues(wellboreTrajectory->getXyzPointCountOfAllPatches());
-			std::unique_ptr<double[]> mdValues = std::make_unique<double[]>(pointCount);
+			std::unique_ptr<double[]> mdValues(new double[pointCount]);
```

Comment:
Not sure about this one.
Is it because you want FESPP to be C++11 compliant?

Reply:
Actually it must be C++11 more I think about. So I fully agree.

Reply:
I would prefer compile with c++17 but I am not sure of fesapi and paraview...